level-ws.js

```diff
@@ -49,7 +49,7 @@ function WriteStream (options, db) {
     return new WriteStream(options, db)
   }
 
-  Writable.call(this, { objectMode: true })
+  Writable.call(this, { objectMode: true }, write.bind(this), flush.bind(this))
```

Comment:
`{ objectMode: true }` is passed on to `Writeable` that flush stream inherits from.